feature_set_quickstart_guide/test_model_locally.py

```diff
@@ -0,0 +1,31 @@
+from pandas import DataFrame
+from qwak.model.tools import run_local
+
+from main.model import CreditRiskModel
+
+if __name__ == '__main__':
```

Comment:
Awesome, I'd add another file for remote model testing (like we have in the other examples)